httpretty_fixtures/__init__.py

```diff
@@ -87,11 +90,34 @@ def start(cls, fixtures):
                                    'Please invoke `_httpretty_fixtures.mark_fixture` before using `.run()`/`.start()`'
                                    .format(fixture=fixture_key))
 
-            # Update the fixture to know about `self`
-            contextual_fixture = functools.partial(fixture)
-            HTTPretty.register_uri(*fixture._httpretty_fixtures_args, body=contextual_fixture,
+            # Define default information
+            fixture.__func__.first_request = None
+            fixture.__func__.last_request = None
+            fixture.__func__.requests = []
```

Comment:
I should note that this feels janky to me. I also think that this might bleed across server instances but an unsure when that would occur. A better solution might be to create something like a dictionary where we store request info and then allow access via `fake_server.get_fixture_info`. Thoughts?